### DISCUSSION TITLE
taskbar icon

### DISCUSSION THREAD

**Discussion by buraCharles** [valinet/ExplorerPatcher#614] [General]
Which option to only have the icon when the program is started.

![tasbar](https://user-images.githubusercontent.com/93889254/147825949-efcf3959-1b21-4240-be97-2596e0ac5455.JPG)

**Comment by valinet** [comment]
https://github.com/valinet/ExplorerPatcher/issues/612

**Comment by buraCharles** [comment]
thank you.
happy New Year

**Reply by valinet** [reply]
I have issued a hotfix update that addresses this (migrates the previous setting correctly) in the latest release (40.21), so at least users that haven't upgraded yet do not face this anymore. Thanks for the feedback, this release (40.21) contains just that, and it doesn't affect users who have already upgraded (to 40.20, for example), change their settings etc. Again, this happened because I simply haven't thought of this scenario during the time this update was in testing.

Thanks again for the feedback and wish you a great new year!
